lib/manifest.schema.json

```diff
@@ -78,6 +78,14 @@
           },
           "as": {
             "type": "string"
+          },
+          "strategy": {
+            "type": "string",
+            "pattern": "^lazy|beforeInteractive|afterInteractive$"
+          },
```

Comment:
Does `strategy` make sense on CSS?

Reply:
I considered removing it from CSS initially but then decided it was possible that it might be useful to someone in an edge case. However, having now made changes to the default document template, it's starting to look very edge case indeed. To do CSS lazy loading you'd have to use js to build and add a link tag. Maybe thats a use case someone has. Eg. You could use such a trick to load in the styles to match a piece of JS thats also lazy loaded in etc. Doesn't seem harmful to include it. 🤷 Your call, should I leave it or remove it? @trygve-lie

Reply:
Leave it.